Assets/Scripts/Visual Feedback/VisualFeedBackWorker.cs

```diff
@@ -0,0 +1,35 @@
+using JetBrains.Annotations;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VisualFeedBackWorker : MonoBehaviour
+{
+    public GameObject pickaxePrefab;
+    public GameObject pickaxe;
+
+    public float posX = 800f;
+    public float posY = 450f;
+
+    //creates and shows a worker on screen in a row  every time after buying 5 worker 
+    public void MultipleWorker(int workerCount)
+    {
+        if ((workerCount % 5) == 0 && posX < 1070)
+        {
+            //sr.sprite = pickaxe;
+            pickaxePrefab = Resources.Load("pickaxe") as GameObject;
+            pickaxe = GameObject.Instantiate(pickaxePrefab, new Vector2(posX += 45f, posY), transform.rotation);
+            Debug.Log("5 Worker bought" + workerCount);
+        }
+        else if ((workerCount % 5) == 0 && posX >= 1070)
+        {
+            posX = 800f;
+            pickaxePrefab = Resources.Load("pickaxe") as GameObject;
+            pickaxe = GameObject.Instantiate(pickaxePrefab, new Vector2(posX += 45, posY -= 45f), transform.rotation);
+            Debug.Log("5 Worker bought" + workerCount);
+        }
+        if (posY <= 360)
+            Destroy(pickaxe);
+        }
```

Comment:
Diese nicht gut
Alles hardcoded, bedeutet, ist es nicht Full HD, muss man das alles wieder anpassen

Reply:
Ich hab den Code fast komplett umgeschrieben. Ich hoffe, dass es jetzt in Ordnung ist.

Reply:
![image](https://user-images.githubusercontent.com/117286314/206248244-a0f51cc4-be28-4d5d-8b8a-0d1cd2a0fee8.png)

Wofür ist die Scrollbar?
Worker überlappen sich. Horizontal Layout Group --> Images reinziehen --> SetActive(true) im Code | Das wäre zumindest eine Lösung

Reply:
Ich hab die Scrollbar entfernt und ich hab  die Sprite für Full HD optimiert. Aus diesem Grund überlappen die sich. Ich hab   stundenlang versucht eine Lösung zu finden, war aber leider alles umsonst.

Reply:
public void MultipleWorker(int workerCount)
    {
        if ((workerCount % 5) == 0)
        {
            GameObject canvasGameObject = GameObject.Find("Canvas");
            GameObject feedbackWorker = FindObjectHelper.
                FindObjectInParent(canvasGameObject, "FeedbackWorker");
            pickaxePrefab = Resources.Load("pickaxePrefab") as GameObject;
            pickaxe = Instantiate<GameObject>(pickaxePrefab, feedbackWorker.transform);
        }

    }

Reply:
Vielen Dank für die die Hilfe!  ich habe ewig lange probiert und war schon etwas am verzweifeln.